db/schema.rb

```diff
@@ -70,6 +76,7 @@
     t.datetime "created_at",                  null: false
     t.datetime "updated_at",                  null: false
     t.text     "notes"
+    t.integer  "check_in_id"
```

Comment:
It looks like our check_ins table doesn't have an id column? You may need to look at the migration that was used to create teams, as we do want to include an id for each of our check ins.

Further, the check_in_id can't be an integer, since the id's we use in our database are `uuids`; so we'll probably need to change the migration that adds the check_in_id attribute to the logs table to `t.uuid :check_in_id` instead of `t.references :check_in`